### ISSUE TITLE
Фабрики по производству боевых юнитов должны стоить дешевле

### ISSUE THREAD

**Issue by brunneng** [MailRuChamps/raic-2020#97]
Сейчас цена фабрик 500, тогда как юнитов 20, 30 (пусть даже с увеличивающейся стоимостью)
Это значит, что фабрика стоит примерно в 20 раз дороже боевого юнита.
К чему это приведет?
К тому что начиная со 2 раунда, лучшей стратегией будет построить только 1 тип барака и штамповать юниты с него. Барак другого типа построить будет слишком дорого и долго, в это время противник придет с превосходящей на 20 единиц армией, и не оставит шансов. А это значит, что смешанные армии мы не увидим. Причем это 100% будут лучники, т.к. они без проблем размикривают мечников.
Я считаю, что тут организаторы очень неугадали с ценой. Не может фабрика стоить в 20 раз дороже боевого юнита, это дизбаланс.. В том же старкрафте это максимум в 2-3 раза дороже..

Как насчет снизить цену хотя бы до 100 ?

**Comment by boriszaitsev** [comment]
Я поддержу, причем хорошо бы при этом и понизить время строительства.
База сейчас стоит 500, здоровье 300. Расчеты на пальцах для 2 тура:
Первые 100-150 тиков строим рабочих, копим 500 ресов на первый барак. Следующие 100 тиков строим барак. Очень сильно ускорить строительство не получится - надо параллельно добывать ресурс на домики и на солдат, а рабочих у нас 15-20,  обступить стройплощадку со всех сторон не выйдет, так как на картах под 2 тур ресурс часто прилегает вплотную к базе, еще и место под первый барак придется расчищать специально. 
Итого, первые 200-250 тиков все стратегии делают практически одно и то же, боев нет, игра просто сокрацается на четверть. До второго барака действительно дело скорее всего не дойдет.
Я вижу два возможных выхода:
1) сократить стоимость и время постройки бараков, как предлагалось выше. Возможно как-то понерфить сторительство второго и далее барака того же типа (удвоить цену за каждый следующий?).
2) Отсыпать каждому игроку на старте какое-то количество ресурса, сравнимое со стоимостью барака, чтобы строительство базы можно было начинать (почти) сразу.

**Comment by AndrewB330** [comment]
Согласен, 500 как-то слишком много. Правда заметили мы это поздно, после беты) Но в принципе на первый раунд это почти не влияет (снижение цены). Я бы предложил цену 200, это достаточно быстро копится. Ну и по хорошему надо сделать не мгновенное построение юнитов, но это будет слишком заметное изменение.

Здоровье 300 - в принципе терпимо, 10 рабочих за 30 тиков построят
